apps/website/app/utils/getMarkdownFile.ts

```diff
@@ -0,0 +1,50 @@
+import { getHtmlFromMarkdown } from "~/utils/getHtmlFromMarkdown";
+import { getFileContent } from "~/utils/getFileContent";
+import { notFound } from "next/navigation";
+import { PageFrontmatter, PageSchema } from "~/types/schema";
+import matter from "gray-matter";
+
+type Props = {
+  slug: string;
+  directory: string;
+};
+
+type ProcessedMarkdownPage = {
+  data: PageFrontmatter;
+  contentHtml: string;
+};
+
+export const getMarkdownPage = async ({
+  slug,
+  directory,
+}: Props): Promise<ProcessedMarkdownPage> => {
```

Comment:
Are we going to use it in future for other things? 
Otherwise we could also use `BLOG_PATH` directly?

Reply:
yeah for Docs  as you mentioned above